index.html

```diff
@@ -57,9 +57,9 @@ <h2> What is Cryptography? | Introduction to Cryptography | Cryptography for Beg
         <div class="author">
           <div class="author-1">e!</div>
           <p>3.77M Subscribers</p>
-          <button>subscribe</button>
+          <button type="submit">subscribe</button>
           <div class="thumb-icons">
-            <div class="thumb"><i class="fa-solid fa-thumbs-up"></i> <span>|</span> <i
+            <div class="thumb"><i class="fa-solid fa-thumbs-up"></i><span>|</span><i
                 class="fa-solid fa-thumbs-down"></i></div>
             <div><i class="fa-solid fa-share"></i></div>
             <div><i class="fa-solid fa-ellipsis"></i></div>
```

Comment:
Please Changes all element inside this by enclosing them in an unordered list and list items.